kalaponents.variable.inc

```diff
@@ -50,6 +50,23 @@ function kalaponents_variable_info($options = array()) {
     'localize' => FALSE,
   );
 
+  // This multi-variable* holds the template paths for each component type.
+  $variable['kalaponents_templatepath_[type]'] = array(
```

Comment:
This looks perfect. One thing: I know I used `templatepath` in the demo when we were talking. But now that I see I see it written out, it looks funny to me. How do you feel about changing it to just `template`?